test/test_distributed.py

```diff
@@ -27,6 +28,12 @@
 except ImportError:
     HAS_TORCHVISION = False
 
+CPP_EXTENSIONS_WARNING = """
+Ninja (https://ninja-build.org) must be available to run C++ extensions tests,
+but it could not be found. Install ninja with `pip install ninja`
+or `conda install ninja`. Alternatively, disable the test with
+`run_test.py --exclude c10d`.
```

Comment:
This comment is no longer up to date with the exclusion comment.

I think you can remove the "disable" comment because disabling the whole distributed test just for missing ninja is a bit much in my opinion. Having the message show that the test is skipped should be enough.

Reply:
removed